debian/copyright

```diff
@@ -1,59 +1,781 @@
+ sql/sql_priv.h
+ sql/sql_profile.h
+ sql/sql_reload.h
+ sql/sql_rename.h
+ sql/sql_repl.h
+ sql/sql_schema.h
+ sql/sql_select.h
+ sql/sql_sequence.h
+ sql/sql_servers.h
+ sql/sql_show.h
+ sql/sql_signal.h
+ sql/sql_sort.h
+ sql/sql_statistics.h
+ sql/sql_string.h
+ sql/sql_table.h
+ sql/sql_test.h
+ sql/sql_time.h
+ sql/sql_trigger.h
+ sql/sql_truncate.h
+ sql/sql_tvc.h
+ sql/sql_type_fixedbin.h
+ sql/sql_type_fixedbin_storage.h
+ sql/sql_type_geom.h
+ sql/sql_type.h
+ sql/sql_type_int.h
+ sql/sql_type_json.h
+ sql/sql_type_real.h
+ sql/sql_type_string.h
+ sql/sql_udf.h
+ sql/sql_union.h
+ sql/sql_update.h
+ sql/sql_view.h
+ sql/sql_window.h
+ sql/strfunc.h
+ sql/structs.h
+ sql/sys_vars_shared.h
+ sql/table_cache.h
+ sql/table.h
+ sql/thread_cache.h
+ sql/threadpool_generic.h
+ sql/threadpool.h
+ sql/threadpool_winsockets.h
+ sql/thr_malloc.h
+ sql/transaction.h
+ sql/tzfile.h
+ sql/tztime.h
+ sql/uniques.h
+ sql/unireg.h
+ sql/vers_string.h
+ sql/winservice.h
+ sql/wsrep_allowlist_service.h
+ sql/wsrep_applier.h
+ sql/wsrep_binlog.h
+ sql/wsrep_client_service.h
+ sql/wsrep_client_state.h
+ sql/wsrep_condition_variable.h
+ sql/wsrep_high_priority_service.h
+ sql/wsrep_mutex.h
+ sql/wsrep_mysqld_c.h
+ sql/wsrep_mysqld.h
+ sql/wsrep_on.h
+ sql/wsrep_priv.h
+ sql/wsrep_schema.h
+ sql/wsrep_server_service.h
+ sql/wsrep_server_state.h
+ sql/wsrep_sst.h
+ sql/wsrep_status.h
+ sql/wsrep_storage_service.h
+ sql/wsrep_thd.h
+ sql/wsrep_trans_observer.h
+ sql/wsrep_types.h
+ sql/wsrep_utils.h
+ sql/wsrep_var.h
+ sql/wsrep_xid.h
+ sql/xa.h
+ strings/ctype-mb.h
+ strings/ctype-simple.h
+ strings/ctype-ucs2.h
+ strings/ctype-unicode1400-casefold.h
+ strings/ctype-unicode1400-casefold-tr.h
+ strings/ctype-unicode300-casefold.h
+ strings/ctype-unicode300-casefold-tr.h
+ strings/ctype-unicode300-general_ci.h
+ strings/ctype-unicode300-general_mysql500_ci.h
+ strings/ctype-unicode520-casefold.h
+ strings/ctype-unidata.h
+ strings/ctype-utf16.h
+ strings/ctype-utf32.h
+ strings/ctype-utf8.h
+ strings/strings_def.h
+ tpool/tpool.h
+ tpool/tpool_structs.h
+ vio/vio_priv.h
+Copyright: 1995-2008, MySQL AB, 2009 Sun Microsystems, Inc.
+  2000-2019, Oracle and/or its affiliates.
+  2003-2007, MySQL AB, 2009 Sun Microsystems, Inc.
+  2006-2008, MySQL AB, 2008 Sun Microsystems, Inc.
+  2006, MySQL AB, 2009 Sun Microsystems, Inc.
+  2007-2008, Sun Microsystems, Inc,
+  2007, Google Inc.
+  2007, MySQL AB, 2008 Sun Microsystems, Inc.
+  2008, 2011, 2013 SkySQL Ab.
+  2008-2024, MariaDB Corporation
+  2008-2023, Codership Oy <http://www.codership.com>
+  2008, MySQL AB
+  2008, MySQL AB, 2008-2009 Sun Microsystems, Inc.
+  2008, Sun Microsystems, Inc.
+  2009-2011, 2013, 2017, 2020, Monty Program Ab
+  2013, Kristian Nielsen and MariaDB Services Ab.
+  2013, Sergei Golubchik and Monty Program Ab
+  2013, Sergey Vojtovich and MariaDB Foundation
+  2014-2015, SkySQL Ab & MariaDB Foundation
+  2014, Google Inc.
+  2014, SkySQL Ab, MariaDB Corporation
+  2015, Daniel Black.
```

Comment:
Daniel Black is an independent author of all these files in 2015? Surely he would have signed a CLA.